### PR TITLE
Add `LC_ALL` and `LANG` export for Amazon Linux 2

### PR DESCRIPTION
**Pull request by amenasria**
### What does this PR do?

This PR adds 
```
export LC_ALL=en_US.UTF-8
export LANG=en_US.UTF-8
```
to the installation troubleshooting section because it will raise an error elseway.

### Motivation

The instructions for Amazon Linux 2 need:
```
export LC_ALL=en_US.UTF-8
export LANG=en_US.UTF-8
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,21 @@ depends 'yum', '< 5.0'
...
+ValueError: unknown locale: UTF-8
+```
+
+You can fix this by running:
+```
+export LC_ALL=en_US.UTF-8
+export LANG=en_US.UTF-8
+```
+
+You can now restart the installation and it should be fine.
```
Comment (alai97) [inline README.md:109]:
```suggestion
Then, restart the installation.
```

File: README.md [hunk 2]
```diff
@@ -93,6 +93,21 @@ depends 'yum', '< 5.0'
 
 5. Wait for the next scheduled `chef-client` run or trigger it manually.
 
+#### Troubleshooting
+
+On some OS (for example Amazon Linux 2) you might have the following error during installation:
```
Comment (alai97) [inline README.md:98]:
```suggestion
On some operating systems such as Amazon Linux 2, you may encounter the following error during installation:
```

**Review by alai97 — APPROVED** [review]
Looks good, two suggestions!

**Comment by amenasria** [comment]
Thanks for the review ! Closing the PR still because we'll finally implement the script directly in the recipe which will make it way cleaner.

**Pull request closed without merging**